website/docs/d/ip_ranges.html.markdown

```diff
@@ -25,8 +25,8 @@ resource "aws_security_group" "from_europe" {
     from_port        = "443"
     to_port          = "443"
     protocol         = "tcp"
-    cidr_blocks      = ["${data.aws_ip_ranges.european_ec2.cidr_blocks}"]
-    ipv6_cidr_blocks = ["${data.aws_ip_ranges.european_ec2.ipv6_cidr_blocks}"]
+    cidr_blocks      = "${data.aws_ip_ranges.european_ec2.cidr_blocks}"
+    ipv6_cidr_blocks = "${data.aws_ip_ranges.european_ec2.ipv6_cidr_blocks}"
```

Comment:
Hi @bigwheel 👋 Thank you for fixing this for Terraform 0.12 syntax. To prevent any confusion for operators still on Terraform 0.11, can you please also remove the string interpolation syntax?

```md
    cidr_blocks      = data.aws_ip_ranges.european_ec2.cidr_blocks
    ipv6_cidr_blocks = data.aws_ip_ranges.european_ec2.ipv6_cidr_blocks
```